### PR TITLE
Update security.rst

### PR DESCRIPTION
**Pull request by abdounikarim**
Hello,

I just updated the link which was wrong.

### CODE REVIEW

File: security.rst [hunk 1]
```diff
@@ -27,7 +27,7 @@ creates a ``security.yaml`` configuration file for you:
 
     # config/packages/security.yaml
     security:
-        # https://symfony.com/doc/current/security/experimental_authenticators.html
+        # https://symfony.com/doc/current/security/authenticator_manager.html
```
Comment (javiereguiluz) [inline security.rst:30]: The old URL just redirects to the main security page ... but the proposed new URL returns a 404 error. @wouterj would it be better to just remove this URL? Thanks.

Comment (abdounikarim) [reply security.rst:30]: @javiereguiluz and @wouterj i just updated from here : https://github.com/symfony/recipes/blob/master/symfony/security-bundle/5.3/config/packages/security.yaml

So, we need to update this file too.

Comment (javiereguiluz) [reply security.rst:30]: You are right. I'm proposing to update some links in https://github.com/symfony/recipes/pull/1011

**Review by 94noni — APPROVED** [review]

**Comment by javiereguiluz** [comment]
@abdounikarim I'm really sorry but I must close this PR without merging because we recently revamped all the security docs (see #15503) and we removed that link, so there's no need to fix it anymore. Thanks!

**Pull request closed without merging**